index.bs

```diff
@@ -243,6 +243,50 @@ scenarios. Additional scenarios, including sample code, are given later in [[#sa
     * Web page shows that the selected user is signed in, and navigates to the signed-in page.
 
 
+### New device enrollment ### {#usecase-new-device-enrollment}
+
+This use case scenario illustrates how a [=[RP]=] can leverage a combination of a [=roaming authenticator=] (e.g., a USB security
+key fob) and a [=platform authenticator=] (e.g., a built-in fingerprint sensor) such that the user has:
+
+  - a "primary" [=authenticator=] that they use to authenticate on new-to-them devices (e.g., laptops, desktops) or on such
+    devices that lack a [=platform authenticator=], and
+  - a low-friction means to strongly re-authenticate on devices having [=platform authenticators=].
+
+Note: This approach of registering multiple [=authenticators=] for an account is also useful in account recovery use cases.
+
+Note: The various prompts and user-entered texts in this section are examples, and not normative.
+
+- First, on a desktop computer (lacking a [=platform authenticator=]):
+    * User navigates to `example.com` in a browser and signs in to an existing account using whatever method they have been using
+        (possibly a legacy method such as a password), or creates a new account.
+    * User navigates to account security settings and selects "Register security key".
+    * Website prompts the user to plug in a USB security key fob and enter a nickname for the key.
+    * User enters "Keychain key" and plugs in the USB security key.
+    * The USB security key blinks to indicate the user should press the button on it; the user does.
+    * Website shows message, "Registration complete."
+
+    Note: Since this computer lacks a [=platform authenticator=], the website may require the user to present their USB security
+    key from time to time or each time the user interacts with the website. This is at the website's discretion.
+
+- Later, on their laptop (which features a [=platform authenticator=]):
+    * User navigates to example.com in a browser and initiates signing in.
+    * Website prompts the user to plug in the "Keychain key" USB security key.
+    * User plugs in the previously registered USB security key and presses the button.
+    * Website shows that the user is signed in, and navigates to the signed-in page.
+    * Website prompts, "Do you want to register this computer with example.com?"
+    * User agrees.
+    * Laptop prompts the user for a previously enrolled [=authorization gesture=] (PIN, biometric, etc.); the user provides this.
+    * Website prompts the user for a device nickname; user enters "Work laptop".
+    * Website shows message, "Registration complete."
+    * User signs out.
+
+- Later, again on their laptop:
+    * User navigates to example.com in a browser and initiates signing in.
+    * Website shows message, "Please follow the instructions in your browser to sign in."
```

Comment:
In this username less flow, even if platform finds a key for that RP, Platform needs to provide a way to support external FIDO devices in the UI as platform does not know whether user has multiple keys associated with that RP and wants to use external keys for whatever reason.

Reply:
In with username flow, where platform gets credential ID allow list, if all the credential ID passed are found on the authenticator, platform can skip provide an option for the user to user external devices. If not all the credentialID are found on the platform, but some are found, again, platform needs to provide an option to the user to use external keys.

Reply:
Yes, this is true, but I don't think the example needs to exhaustively detail all the possibilities. I think it'll be clearer to just illustrate one of the possible ways it could play out. What do you think?

Reply:
perhaps add a Note indicating that there are variations to the above if a user has registered a mix of platform and roaming authnrs with a given webapp?